crates/redstone_testing/src/scenario.rs

```diff
@@ -0,0 +1,269 @@
+use core::time::Duration;
+
+use redstone::Value;
+
+use crate::{
+    env::{PriceAdapterRunEnv, Signer},
+    sample::{Sample, Signers, DEFAULT_SIGNERS_THRESHOLD},
+};
+
+enum Action {
```

Comment:
maybe move to a separate file?

Reply:
this I would like to be here as its only used by Scenario internals